airflow/gcp/hooks/gcs.py

```diff
@@ -405,11 +420,9 @@ def create_bucket(self,
         """
         Creates a new bucket. Google Cloud Storage uses a flat namespace, so
         you can't create a bucket with a name that is already in use.
-
         .. seealso::
             For more information, see Bucket Naming Guidelines:
             https://cloud.google.com/storage/docs/bucketnaming.html#requirements
-
```

Comment:
This change prevents the documentation from being built properly. Can you delete it?